cfgov/v1/tests/test_wagtail_hooks.py

```diff
@@ -118,3 +117,49 @@ def test_calls_child_block_is_submitted(self, mock_getstreamfields, mock_hasattr
         form_module_handlers(self.page, self.request, self.context)
         child.block.is_submitted.assert_called_with(self.request, 'name', 0)
 
+
+class TestRelativePageLinkHandler(TestCase):
+    def setUp(self):
+        self.default_site = Site.objects.get(is_default_site=True)
+
+    def test_nonexisting_page_returns_empty_link(self):
+        self.assertEqual(
+            RelativePageLinkHandler.expand_db_attributes(
+                attrs={'id': 0},
+                for_editor=False
+            ),
+            '<a>',
+        )
+
+    def test_page_not_in_site_root_returns_none_link(self):
+        self.assertEqual(
+            RelativePageLinkHandler.expand_db_attributes(
+                attrs={'id': 1},
+                for_editor=False
+            ),
+            '<a href="None">',
```

Comment:
I kept this behavior [consistent with how Wagtail does it](https://github.com/wagtail/wagtail/blob/master/wagtail/wagtailcore/rich_text.py#L50). To clarify, it's not "the site root", it's "any site root" - meaning this only appears if a link is to a page that isn't associated with any `Site` at all (wouldn't even have an absolute URL).